src/xamarin/SkiaKeeper.c

```diff
@@ -68,6 +69,10 @@ void** KeepSkiaCSymbols (void)
         (void*)sk_bitmap_new,
         (void*)sk_canvas_destroy,
         (void*)sk_codec_min_buffered_bytes_needed,
+        (void*)sk_codec_new_from_data,
+        (void*)sk_codec_new_from_stream,
+        (void*)sk_codec_new_from_stream_with_png_chunk_reader_and_selection_policy,
+        (void*)sk_codec_new_from_data_with_png_chunk_reader,
```

Comment:
This should not be needed as there is already a item from that sk_codec file.

Reply:
I like to have them just in-case

Reply:
Not sure if it is required or not in order to avoid blah cannot be found in blah native library or something

Reply:
All that is needed is one symbol from each file to prevent the linker removing it.